src/Microsoft.VisualStudio.SlowCheetah/Resources/Resources.resx

```diff
@@ -125,4 +129,24 @@
     <value>Transform file not found</value>
     <comment>Message to be shown when the transform file is not found</comment>
   </data>
+  <data name="ErrorMessage_UnsupportedFile" xml:space="preserve">
+    <value>{0} is not a supported file type for transformation</value>
+  </data>
+  <data name="JsonTransform_TransformFileContents" xml:space="preserve">
+    <value>{
+}
+</value>
+  </data>
+  <data name="XmlTransform_ContentInfo" xml:space="preserve">
+    <value>For more information on using transformations see the web.config examples at http://go.microsoft.com/fwlink/?LinkId=214134. </value>
+    <comment>Info on XDT transforms, to be added as a comment in the transform file</comment>
+  </data>
+  <data name="XmlTransform_XdtAttributeName" xml:space="preserve">
+    <value>xmlns:xdt</value>
```

Comment:
Don't bother putting non-localized strings in resx like this. Put them as a `const string` in a static class.